src/AppInstallerCommonCore/Public/winget/Experiment.h

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#pragma once
+#include <string>
+#include <type_traits>
+#include "AppInstallerStrings.h"
+
+namespace AppInstaller::Settings
+{
+    enum ExperimentToggleSource
+    {
+        Default = 0,
+        Policy = 1,
+        UserSetting = 2,
+    };
+
+    struct ExperimentState
+    {
+        ExperimentState() = default;
+        ExperimentState(bool isEnabled, ExperimentToggleSource toggleSource) : m_isEnabled(isEnabled), m_toggleSource(toggleSource) {}
+        bool IsEnabled() const { return m_isEnabled; }
+        ExperimentToggleSource ToggleSource() const { return m_toggleSource; }
+        std::string ToJson() const;
+    private:
+        ExperimentToggleSource m_toggleSource;
+        bool m_isEnabled;
+    };
+
+    struct Experiment
+    {
+        enum class Key : unsigned
+        {
+            None = 0x0,
+            CDN = 0x1,

```

Comment:
Is this going to be one of these enums where we increase by 1 or by powers of 2? I'm guessing the former